android/app/src/main/java/com/on/turip/ui/compose/setting/SettingItem.kt

```diff
@@ -0,0 +1,59 @@
+package com.on.turip.ui.compose.setting
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.on.turip.R
+import com.on.turip.ui.compose.theme.TuripTypography
+
+@Composable
+fun SettingItem(
+    uiModel: SettingModel,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        modifier =
+            modifier
+                .clickable(onClick = onClick)
+                .padding(horizontal = 20.dp, vertical = 14.dp),
+        verticalAlignment = Alignment.CenterVertically,
+    ) {
+        Icon(
+            painter = painterResource(uiModel.iconResource),
+            contentDescription = null,
+            modifier = Modifier.padding(end = 14.dp),
+        )
```

Comment:
```suggestion
        Icon(
            painter = painterResource(uiModel.iconResource),
            contentDescription = null,
            modifier =
                Modifier
                    .padding(end = 14.dp)
                    .size(24.dp),
        )
```
r: 지금 설정화면의 텍스트나 아이콘의 위치가 통일되지 않았는데 size를 통해 Icon 크기를 통일해서 위치를 고정하면 좋을 것 같네요😁

Reply:
좋아요 !
반영 커밋 : 1487bed8ece6f0ea2872475af4420bd55feaaaa5